src/components/Filter/FilterValue.tsx

```diff
@@ -0,0 +1,90 @@
+import { ComponentProps, forwardRef } from 'react';
+import { getCommonProps, px } from '../../utils';
+import classnames from 'classnames';
+import { Text, TextVariants } from '../Text';
+import Input, { InputProps } from '../Input/Input';
+
+// You'll need to change the ComponentProps<"htmlelementname"> to match the top-level element of your component
+export interface FilterValueProps extends Omit<ComponentProps<'div'>, 'onChange' | 'value'> {
+  name: string;
+
+  // Text to be displayed as a label for the neighboring input
+  label: string;
+
+  // Type of input for this filter
+  inputType: 'checkbox' | 'radio';
+
+  // Selected or not
+  isActive: boolean;
+
+  // Should this filter be hidden, most common use will be for view all truncation
+  isHidden?: boolean;
+
+  // Whether or not this filter is disabled
+  disabled?: boolean;
+
+  // Function to trigger when the state of this filter is changed
+  onChange: (e: React.ChangeEvent<HTMLInputElement>) => void;
+
+  inputProps?: InputProps;
```

Comment:
this is a full list of the props that are returned from our form library and will be in the `inputProps` field:
```
export type UseFormRegisterReturn<TFieldName extends InternalFieldName = InternalFieldName> = {
    onChange: ChangeHandler;
    onBlur: ChangeHandler;
    ref: RefCallBack;
    name: TFieldName;
    min?: string | number;
    max?: string | number;
    maxLength?: number;
    minLength?: number;
    pattern?: string;
    required?: boolean;
    disabled?: boolean;
};
```
I think to stop confusion you might just remove `inputProps` and just make all of those `top-level` props with the addition of ones that are not in that response